prefect_docker/containers.py

```diff
@@ -104,3 +104,45 @@ def get_docker_container_logs_flow():
         logs = await run_sync_in_worker_thread(container.logs, **logs_kwargs)
 
     return logs.decode()
+
+
+@task
+async def start_docker_container(
+    container_id: str,
+    docker_host: Optional[DockerHost] = None,
+    **start_kwargs: Dict[str, Any],
+) -> Container:
+    """
+    Start this container. Similar to the docker start command.
+
+    Args:
+        container_id: The container ID to start.
+        docker_host: Settings for interacting with a Docker host.
+        **start_kwargs: Additional keyword arguments to pass to
+            [`client.containers.get(container_id).start`](https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.start).
+
+    Returns:
+        The Docker Container object.
+
+    Examples:
+        Start a container with an ID that starts wtih "c157".
+        ```
+        from prefect import flow
+        from prefect_docker.containers import start_docker_container
+
+        @flow
+        def start_docker_container_flow():
+            container = start_docker_container(container_id="c157")
+            return container
+
+        start_docker_container_flow()
+        ```
+    """
+    logger = get_run_logger()
+
+    with (docker_host or DockerHost()).get_client() as client:
+        logger.info(f"Starting {container_id!r} container.")
+        container = await run_sync_in_worker_thread(client.containers.get, container_id)
```

Comment:
I think it'd be worthwhile to log the whole container ID once we have it.
```suggestion
        container = await run_sync_in_worker_thread(client.containers.get, container_id)
        logger.info(f"Starting container {container.id!r}")
```